### PR TITLE
Accordion - add pad and margin options to theme

### PR DESCRIPTION
**Pull request by britt6612**
#### What does this PR do?
This PR add the option to change how the pad and margin sizes are being applied to the accordion panel heading
#### Where should the reviewer start?
accordionPanel.js
#### What testing has been done on this PR?
storybook
#### How should this be manually tested?
storybook
#### Any background context you want to provide?
When a label was passed into accordionPanel there was no say for the users on what size the margin or pad around it was. This was all done within the AccordionPanel.js starting at Line 70
There was really no way for the users to customize what they wanted for heading. 

The pad was being set to `horizontal: 'xsmall'`
and the margin was being taken from the `Heading line size` for level 4
#### What are the relevant issues?
design system
#### Do the grommet docs need to be updated?
auto
#### Should this PR be mentioned in the release notes?
no
#### Is this change backwards compatible or is it a breaking change?
backwards compatible

### CODE REVIEW

File: src/js/components/AccordionPanel/doc.js [hunk 1]
```diff
@@ -21,6 +21,16 @@ export const themeDoc = {
     type: 'number',
     defaultValue: '4',
   },
+  'accordion.heading.margin': {
+    description: 'The margin around heading in the accordion.',
+    type: 'string',
+    defaultValue: 'undefined',
+  },
+  'accordion.heading.pad': {
```
Comment (ShimiSun) [inline src/js/components/AccordionPanel/doc.js:29]: Semantically, the pad isn't assigned to heading, so I think it should be accordion.pad

File: src/js/components/AccordionPanel/doc.js [hunk 2]
```diff
@@ -21,6 +21,16 @@ export const themeDoc = {
...
+    description: 'The pad size around heading in the accordion.',
+    type: 'sting',
+    defaultValue: 'horizontal: xsmall',
```
Comment (ShimiSun) [inline src/js/components/AccordionPanel/doc.js:32]:
```suggestion
    defaultValue: '{ horizontal: xsmall }',
```

File: src/js/components/AccordionPanel/doc.js (hunk 1 above)
Comment (britt6612) [reply src/js/components/AccordionPanel/doc.js:29]: I saw that as it was on `box` Ill change thanks

Comment (ShimiSun) [reply src/js/components/AccordionPanel/doc.js:29]: what about fixing it on the docs as well?

**Comment by ShimiSun** [comment]
Can you please fix this PR? The Changed file doesn't make sense and I suspect its a merge issue.

**Pull request closed without merging**